desktop/renderer-app/src/stores/WhiteboardStore.ts

```diff
@@ -149,6 +150,19 @@ export class WhiteboardStore {
                     if (modifyState.broadcastState) {
                         this.updateViewMode(modifyState.broadcastState.mode);
                     }
+
+                    //  preload file when the whiteboard scene state has been changed.
+                    if (room.state.sceneState.scenes[0].ppt) {
```

Comment:
why is `scenes[0]`? why can't you use 1 or other numbers ?

Reply:
scenes[0] is first ppt page